Source/PINDiskCache.m

```diff
@@ -1106,7 +1106,10 @@ - (void)enumerateObjectsWithBlock:(PINDiskCacheFileURLBlock)block
             NSURL *fileURL = [self encodedFileURLForKey:key];
             // If the cache should behave like a TTL cache, then only fetch the object if there's a valid ageLimit and  the object is still alive
             if (!self->_ttlCache || self->_ageLimit <= 0 || fabs([[_dates objectForKey:key] timeIntervalSinceDate:now]) < self->_ageLimit) {
-                block(key, fileURL);
+                BOOL stop;
+                block(key, fileURL, &stop);
+                if (stop)
+                  break;
```

Comment:
super nit, but can you do 4 spaces here?

Reply:
Done.

Reply:
I would recommend creating a .clang-format file containing your specific style rules. It works great with objc, and you could probably integrate it as a presubmit check on your CI.

See https://clang.llvm.org/docs/ClangFormat.html.